libretro-buildbot-recipe.sh

```diff
@@ -750,9 +763,11 @@ buildbot_pull(){
 					cd $DIR
 					git checkout "$BRANCH"
 				elif [ ! -z "$GIT_BRANCH" ]; then
-					git clone -b "$GIT_BRANCH" "$URL" "$DIR" --depth=1
+					git clone -b "$GIT_BRANCH" "$URL" "$DIR" --depth=2
+					buildbot_tag
 				else
-					git clone -b master "$URL" "$DIR" --depth=1
+					git clone -b master "$URL" "$DIR" --depth=2
+					buildbot_tag
```

Comment:
Actually its been a while and I should of read this closer, the changes should not be done here. This is only for RetroArch assets, this won't help any core is pointless here.

Take a look at the linux retroarch-linux-x64.ra recipe file for instance.

https://github.com/libretro/libretro-super/blob/master/recipes/linux/retroarch-linux-x64.ra

With this change it will start trying to pointlessly add tags (I assume they don't exist?) for libretrodb, shaders, assets and etc.

Reply:
Hi, I actually did double check this and although you're right, I still think it makes sense to have it. It will not impact anything as the assets files are saved without any git information.

But if a core has no branch definition, and points to master, this may still be useful.

What do you think?

Reply:
No cores are going to use the `buildbot_pull` function, only a few specific repos that will never need or use tags.

Ideally it would best to not handle RetroArch and core repos under different parts of the script, but I never got around to cleaning that up...

Reply:
Ok I'll remove the line that, if you give me this assurance. All good :)

Reply:
Done

Reply:
I don't think you understood. You should not alter the `buildbot_pull` function in this PR at all. It doesn't do anything for any core.